src/data/validate_simulator.py

```
@@ -520,6 +520,30 @@ def main() -> None:
+        sys.exit(1)
+    print(f"[PASS] T/C ratio: {tc_ratio:.1%}")
+
+    # 9. 8개 이벤트 타입 존재 여부 검증
+    expected_events = {"page_view", "search", "add_to_cart", "remove_from_cart",
+                       "purchase", "coupon_use", "review", "cs_contact"}
+    actual_events = set(events["event_type"].unique())
+    missing = expected_events - actual_events
+    if missing:
+        print(f"[FAIL] missing event types: {missing}")
+        sys.exit(1)
+    print(f"[PASS] all 8 event types present")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)